<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_statistic</class>
 <widget class="QWidget" name="form_statistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Статистика</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Статистика</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Студент:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>55</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Учитель:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Группа:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_student">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_teacher">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>55</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_group">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Задание 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
